src/Sulu/Component/Content/Compat/Serializer/PageBridgeHandler.php

```diff
@@ -23,7 +27,7 @@
 /**
  * Handle serialization and deserialization of the PageBridge.
  */
-class PageBridgeHandler implements SubscribingHandlerInterface
+class PageBridgeHandler implements SubscribingHandlerInterface, EventSubscriberInterface
```

Comment:
Does mixing both interfaces feel right to you? Confused me at first :thinking:

Reply:
Done created an own PageBridgeSubscriber